### PR TITLE
Notiwire integration

### LINKED ISSUE #34: Integration with notiwire
Online Notifier uses data from [notiwire](https://github.com/appKom/notiwire) to display information about coffee etc.  https://passoa.online.ntnu.no/api/affiliation/abakus
At the moment Abakus' coffee data is cached with an unfortunate long timeout. I've started working on a better solution for this, but that's another story.

The question is really if you would be willing to integrate with our coffee notification API? This way the cache is automatically purged and there will be no need to keep polling your API. 
It's a simple [endpoint](https://github.com/appKom/notiwire/wiki/notipi#coffee) that you would send a single POST request to when a new coffee pot is ready.

I can submit a PR if there is any interest in this. :)

### PR DESCRIPTION
**Pull request by duvholt**
I noticed that you also had an open issue regarding Slack integration so I took the liberty to add a super simple integration framework.

Fixes  #34 

You need an API key for this to work. Who should I send it to?

### CODE REVIEW

File: coffee/integrations.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    API_URL = 'https://passoa.online.ntnu.no/notipi/abakus/coffee'
+    headers = {
+        'User-Agent': 'Abakus Coffee'
+    }
+
+    def __init__(self, api_key):
+        self.api_key = api_key
+
+    def notify(self):
+        requests.post(self.API_URL, data={'api_key': self.api_key}, headers=self.headers)
```
Comment (eirsyl) [inline coffee/integrations.py]: Should we do some exception handling here?

**Comment by eirsyl** [comment]
Nice work! You can send the api key to eirik@sylliaas.no ! 😄

File: coffee/config.py [hunk 2]
```diff
@@ -9,3 +10,7 @@
     'SERVER_HOST': os.getenv('SERVER_HOST', '127.0.0.1'),
     'SERVER_PORT': int(os.getenv('SERVER_PORT', 5000))
 }
+
+integrations = [
+    NotiwireIntegration(os.getenv('NOTIWIRE_KEY', 'invalid_key'))
```
Comment (duvholt) [inline coffee/config.py]: Good point!

File: coffee/integrations.py (hunk 1 above)
Comment (duvholt) [inline coffee/integrations.py]: Yeah, should I add some logging?

**Pull request merged**